k8s/krsync

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# https://serverfault.com/questions/741670/rsync-files-to-a-kubernetes-pod
+#  example usage:
+# ./krsync -av --progress --stats <hostdir>  podName@namespace:<poddir>
+
+if [ -z "$KRSYNC_STARTED" ]; then
+    export KRSYNC_STARTED=true
+    exec rsync -a --delete --blocking-io --rsh "$0" $@
+fi
+
+# Running as --rsh
+namespace=''
+pod=$1
+shift
+
+# If user uses pod@namespace, rsync passes args as: {us} -l pod namespace ...
+if [ "X$pod" = "X-l" ]; then
+    pod=$1
+    shift
+    namespace="-n $1"
+    shift
+fi
+
+# customise this to cluster needs!!
```

Comment:
What sort of customisation is expected here?

Reply:
loose comment when i first did have now removed